### ISSUE TITLE
Should ROBOT report somehow ignore COB top level classes from the "no superclass" check?

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBO-Dashboard#52]
There is actually a technical problem with migrating Marks approach to ODK, which is just a note to self: ROBOT report would, in an ODK like environment, complain about the COB classes themselves to not have parents, because they have been merged into the reasoned-reduced-relaxed file... But its a minor issue that could be addressed on the ROBOT report side, but probably we need some more bespoke COB qc method here rather than robot report. Not sure.

**Comment by bpeters42** [comment]
I thought that in general, for future work we will have separate COB specific checks integrated into the OBO dashboard (or potentially as a separate suite). Those would essentially be the principle checks for "Commitment To Collaboration (principle 10)' and potentially 'Scope'.

**Comment by matentzn** [comment]
Yes probably thats better, just wanted to note that this may confuse dashboard users..

**Comment by allenbaron** [comment]
_A little tangential but related to this discussion:_
It might be nice to be able to specify classes that can be ignored by the dashboard (or better, in my opinion, ROBOT report) for the missing superclass check. This specific check prevents the OBO dashboard and Summary columns from ever being green even when an ontology passes all the tests when the top-level term doesn't have a declared superclass (see [trans](http://dashboard.obofoundry.org/dashboard/trans/dashboard.html) as an example).

**Comment by matentzn** [comment]
@allenbaron you are right - I personally think we should exclude the "no superclass check" entirely from the OBO Dashboard for this very reason!
